mobile/src/components/FooterNav/navigator.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import { NewsFlow, TabBar, WatchFlow } from "./TabBar";
+import Home from "../../screens/Home";
+import { ParamListBase, RouteProp } from "@react-navigation/native";
+
+const Tab = createBottomTabNavigator();
+
+export const BottomTabs = function (props: {
+    route: RouteProp<ParamListBase, string>;
+    navigation: any;
+    options: any;
+}) {
+    return (
+        <Tab.Navigator
+            screenOptions={props.options}
+            tabBar={(props) => <TabBar {...props} />}>
+            <Tab.Screen name="Home" component={Home} />
+            <Tab.Screen name="News" component={NewsFlow} />
+            <Tab.Screen name="Watch" component={WatchFlow} />
```

Comment:
@Zakpak0 This is exactly what I was looking for. I started working through this migration and I'll probably open a PR so you can see the new structure, but this is 🔥!

Reply:
👍

Reply:
Hey @christoph-codes were you having success with this? I wanted to verify it was working since I couldn't get the app to run without access to your backend.

Reply:
@Zakpak0 Yes I got this working. opening a PR now that I will tag you in to see how I structured everything but I needed a combination of both that tab navigator and the stack navigator.